v20.1/cockroachcloud-upgrade-to-v20.1.md

```diff
@@ -0,0 +1,161 @@
+
+Note that these limitations are specific to upgrades from v19.2 to v20.1; they does not apply to other upgrades.
+
+## Step 4. Start the upgrade
+
+Once you have completed your preparations, starting the upgrade process is simple:
+
+1. [Sign in](https://cockroachlabs.cloud/) to your CockroachCloud account.
+
+2. In the **Clusters** list, select the cluster you want to upgrade.
+
+3. Select **Actions > Upgrade cluster**.
+
+4. On the **Upgrade your cluster** dialog, confirm that you have review the pre-upgrade guidance and then click **Start Upgrade**.
+
+<section class="filter-content" markdown="1" data-scope="multi-node">
+As mentioned earlier, your cluster will be upgraded one node at a time without interrupting the cluster's overall health and availability.
+</section>
+
+<section class="filter-content" markdown="1" data-scope="single-node">
+As mentioned earlier, your single-node cluster will be briefly unavailable while the node is stopped and restarted with v20.1.
+</section>
+
+## Step 5. Monitor the upgrade
+
+Once your cluster is running v20.1, you will have approximately 24 hours before the upgrade is automatically finalized. During this time, it is important to monitor your application and respect temporary limitations.
+
+### Monitor your application
+
+Use the [Admin UI](cockroachcloud-monitoring-page.html) or your own tooling to monitor your application for any unexpected behavior.
+
+- If everything looks good, you can wait for the upgrade to automatically finalize or you can [trigger finalization more quickly](#finalize-the-upgrade).
+
+- If you see unexpected behavior, you can [rollback to v19.2](#roll-back-the-upgrade). This option is available only during the 24-hour window. If you see unexpected behavior after the upgrade has been finalized, you will have to [reach out to support](https://support.cockroachlabs.com/hc/en-us/requests/new).
+
+### Respect temporary limitations
+
+Before your upgrade has been finalized, remember to prevent new schema changes and changes to user privileges as mention [earlier](#review-temporary-limitations).
+
+Also, most v20.1 features can be used right way, but there are some that will be enabled only after the upgrade has been finalized. Attempting to use these features before then will result in errors:
+
+- **Primary key changes:** After finalization, it will be possible to change the primary key of an existing table using the [`ALTER TABLE ... ALTER PRIMARY KEY`](alter-primary-key.html) statement, or using [`DROP CONSTRAINT` and then `ADD CONSTRAINT`](add-constraint.html#drop-and-add-a-primary-key-constraint) in the same transaction.
```

Comment:
Same here; we do not get directed to the "drop-and-add-a-primary-key-constraint" section on `add-constraint.html`.

Reply:
OK. Using full urls throughout in those cases. Very annoying, but let's see if it solves the issue.